docs/on_failure.md

```diff
@@ -0,0 +1,15 @@
+# Fixing Issues with Discovering A Supported Container Runtime Environment
+
+If you ended up on this page, 
+it seems that either Testcontainers was not able to find a supported container runtime in your environment,
+or you found page while browsing the documentation in search of information on dealing with errors regarding Testcontainers' environment discovery mechanism.
+
+Testcontainers requires a supported container runtime environment to be present in order to function.
+Here is a list of supported container runtime environments, that can get you up to speed quickly:
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+* [Docker Engine on Linux](https://docs.docker.com/engine/install/)
+* [Testcontainers Cloud](https://www.testcontainers.cloud)
+
+For more extensive information on supported container runtime environments, and how to configure container runtime environments that are not supported by default,
```

Comment:
For more extensive information on supported container runtime environments, as well as known limitations of alternative container runtime environments, please refer to...